crates/web5/src/lib.rs

```diff
@@ -4,3 +4,30 @@ pub mod dids;
 
 #[cfg(test)]
 mod test_helpers;
+
+lazy_static::lazy_static! {
+  pub(crate) static ref LOG_LEVEL: Option<String> = {
+      std::env::var("WEB5_SDK_LOG_LEVEL").ok()
+  };
+}
+
+pub(crate) mod logging {
```

Comment:
could we create an Issue on the backlog to analyze all the relevant areas where debug logging makes sense and subsequently add it? this is a great starting point but we should also _use_ the things

Reply:
for sure
https://github.com/TBD54566975/web5-rs/issues/287